components/access-management/main.go

```diff
@@ -28,22 +28,26 @@ const USERIDHEADER = "userid-header"
 const USERIDPREFIX = "userid-prefix"
 // set cluster admin user id here.
 const CLUSTERADMIN = "cluster-admin"
+// identity used for workload-identity plugin.
+const WORKLOADIDENTITY = "workload-identity"
```

Comment:
Default value for WORKLOADIDENTITY should be disabled ("").
Having it enabled by default assumes everyone will use GCP by default.

Reply:
The current default arg value is "", WORKLOADIDENTITY is off by default.